include/onnxruntime/core/framework/allocator.h

```diff
@@ -101,7 +101,9 @@ class IAllocator {
   const OrtMemoryInfo& Info() const { return memory_info_; };
 
   // Each implementation of IAllocator can override and provide their own implementation
-  virtual void GetStats(AllocatorStats* /*stats*/) { return; }
+  virtual void GetStats(AllocatorStats* stats) {
+    stats->Clear();
```

Comment:
```suggestion
  virtual void GetStats(AllocatorStats& stats) {
    stats.Clear();
```

Use a reference if the argument is required to clearly advertise what the API wants.

Reply:
if it fans out to a lot of files it's fine to leave.